src/antsibull_nox/config.py

```diff
@@ -173,6 +175,63 @@ class SessionLicenseCheck(_BaseModel):
     license_check_extra_ignore_paths: list[str] = []
 
 
+class ExecutionEnvironmentConfig(_BaseModel):
+    """
+    Execution enviroment check session config.
+    """
+
+    name: str
+    description: t.Optional[str] = None
+    test_playbooks: list[str]
+    version: int = 3
```

Comment:
I'd restrict the possible values of `version` to `3`, since the code currently doesn't allow anything else:
```suggestion
    version: t.Literal[3] = 3
```